lib/cryptoexchange/exchanges/bitmesh/services/pairs.rb

```diff
@@ -0,0 +1,32 @@
+module Cryptoexchange::Exchanges
+  module Bitmesh
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Bitmesh::Market::API_URL}"
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          pairs = output
+          market_pairs = []
+          pairs["data"].each do |pair|
+            unless pair[1]['price'].nil?
```

Comment:
some ticker does not return `last`, so to exclude these tickers. example as below:-

```
        "eth_bch": {
            "name": "eth_bch",
            "change": 0
        },
```